README.md

```diff
@@ -1,24 +1,32 @@
-LLVM "Nightly Test" Infrastructure
-==================================
+LNT: LLVM Nightly Test Infrastructure
+=====================================
 
-This directory and its subdirectories contain the LLVM nightly test
-infrastructure. This is technically version "4.0" of the LLVM nightly test
-architecture.
+About
+-----
 
-The infrastructure has the following layout:
+LNT is an infrastructure for performance testing. The software itself
+consists of two main parts, a web application for accessing and visualizing
+performance data, and command line utilities to allow users to generate and
+submit test results to the server.
 
- $ROOT/lnt - Top-level Python 'lnt' module
+The package was originally written for use in testing LLVM compiler
+technologies, but is designed to be usable for the performance testing of any
+software.
 
- $ROOT/lnt/server/db - Database schema, utilities, and examples of the LNT plist format.
 
- $ROOT/docs - Sphinx documentation for LNT.
+Documentation
+-------------
 
- $ROOT/tests - Tests for the infrastructure.
+The official *LNT* documentation is available online at: https://llvm.org/docs/lnt
```

Comment:
Is LNT meant to be italic or bold? Not sure it needs highlighting at all really.

Reply:
Maybe PyPi knows to do something clever with this?